### DISCUSSION TITLE
Can't seem to find Armory Officer role

### DISCUSSION THREAD

**Discussion by ww123ist** [bcholmes/StarTrek2d20#213] [General]
Hello, I can't seem to find "Armory Officer" roles at the final page, which have been described in at least two books thus far. Is there a specific guideline to getting there, or has something happened to where it doesn't appear by accident?

**Comment by bcholmes** [comment]
The Armory Officer role only shows up if either:

1. The Era is "Enterprise" and the character is a Starfleet character (like Malcolm Reed); or
2. The character is an Allied Military character.

**Reply by ww123ist** [reply]
Interesting, however with the new Klingon-Federation war book, it's listed there for a TOS-centered campaign (page 22) which is also adaptable for TNG eras. Do you still think that restriction is necessary? I was hoping to make some sheets with this campaign in mind with this role.
